scanner/templates/laravel/.fly/scripts/caches.sh

```diff
@@ -3,3 +3,9 @@
 /usr/bin/php /var/www/html/artisan config:cache --no-ansi -q
 /usr/bin/php /var/www/html/artisan route:cache --no-ansi -q
 /usr/bin/php /var/www/html/artisan view:cache --no-ansi -q
+
+# Filament v3.2 Performance Improvement with commands 
+if grep "filament/filament.*:.*3\.2" "/var/www/html/composer.json"; then 
```

Comment:
Not sure if you need the options -Fq here as well? 
-F disables regex searching, so would make sense to remove. 
-q makes the command quiet, not sure what impact this would have.

Reply:
removed the options